pronto/api/entries.py

```diff
@@ -243,8 +243,8 @@ def get_recent_entries():
     cur.execute(
         """
         SELECT
-          E.ENTRY_AC, E.ENTRY_TYPE, E.SHORT_NAME, A.TIMESTAMP, A.DBUSER,
-          NVL(U.NAME, A.DBUSER), E.CHECKED, NVL(EC.CNT, 0), NVL(MC.CNT, 0)
+          E.ENTRY_AC, E.ENTRY_TYPE, E.SHORT_NAME, A.TIMESTAMP,NVL(U.NAME, A.DBUSER), 
+          E.CHECKED, NVL(EC.CNT, 0), NVL(MC.CNT, 0), SIGN.SI
```

Comment:
PEP-8 and renaming aliases

```suggestion
          E.ENTRY_AC, E.ENTRY_TYPE, E.SHORT_NAME, A.TIMESTAMP, 
          NVL(U.NAME, A.DBUSER), E.CHECKED, NVL(EC.CNT, 0), NVL(MC.CNT, 0), 
          EM.ACCESSIONS
```